pandas/core/indexes/datetimes.py

```diff
@@ -1706,9 +1706,67 @@ def freq(self, value):
                                   "The microseconds of the datetime")
     nanosecond = _field_accessor('nanosecond', 'ns',
                                  "The nanoseconds of the datetime")
-    weekofyear = _field_accessor('weekofyear', 'woy',
-                                 "The week ordinal of the year")
+    _weekofyear_doc = """
```

Comment:
Ah sorry could you add a trailing backslash to the end of this line? Then we don't have a blank line at the start of the docstring.

Reply:
When I do that the validation script gives an error. 

```
Docstring text (summary) should start in the line immediately after the opening quotes (not in the same line, or leaving a blank line in between)
```

Reply:
Ah, nvm you're good then.